tools/RAiDER/models/gmao.py

```diff
@@ -107,6 +128,7 @@ def _fetch(self, lats, lons, time, out, Nextra=2):
             self._lon_res
         )
 
+
```

Comment:
So would move the try saving to an h5 file part of the GMAO OpenDap call only. 
In future that will be updated to be a netcdf compliant file rather than h5 file.